hw/ip/otbn/doc/_index.md

```diff
@@ -530,6 +530,10 @@ The `URND` LFSR is seeded once from the EDN connected via `edn_urnd` when OTBN s
 Each new execution of OTBN will reseed the `URND` LFSR.
 The LFSR state is advanced every cycle when OTBN is running.
 
+The LFSR-based pseudo-random number generator has a long cycle length but zero is a fixed point for the LFSR, so the sequence of numbers will get stuck at zero if the state ever happens to become zero.
```

Comment:
Does it still use an LFSR? @vrozic

Reply:
Indeed, it is no longer an LFSR. However, zero is still a fixed point.